charts/consul/Chart.yaml

```diff
@@ -22,7 +22,7 @@ annotations:
     - name: consul-dataplane
       image: docker.mirror.hashicorp.services/hashicorppreview/consul-dataplane:1.5-dev
     - name: envoy
-      image: envoyproxy/envoy:v1.25.11
+      image: envoyproxy/envoy:v1.28.2
```

Comment:
We actually don't need to modify this - this is a deprecated value specific to the legacy API gateway, which will go away at some point hopefully soon.

This repo actually doesn't have any need to set the Envoy version explicitly, as it's entirely inherited from `consul-dataplane`.

Reply:
Thanks for the details, that's good to know 👀 I have reverted the value: https://github.com/hashicorp/consul-k8s/pull/3893/commits/cce1e7dbc1660f94cdd568e4282874ce1406e202